.bumpversion.cfg

```
@@ -1,5 +1,5 @@
 [bumpversion]
-current_version = 0.12.4
+current_version = 0.12.5
```

This needs to be a minor release, as the change will be backwards-incompatible.
```suggestion
current_version = 0.13.0
```